### PR TITLE
[1] convert to typescript

### PR DESCRIPTION
**Pull request by rictic**
/cc @cdata

### CODE REVIEW

**Comment by rictic** [comment]
Added travis

File: src/shady-css/common.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+/**
+ * A set of common RegExp matchers for tokenizing CSS.
+ * @constant
```
Comment (justinfagnani) [inline src/shady-css/common.ts]: I think I'd get rid of the JSDoc type annotations...

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "moduleResolution": "node",
+    "isolatedModules": false,
+    "declaration": true,
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "suppressImplicitAnyIndexErrors": false,
+    "removeComments": false,
+    "preserveConstEnums": true,
+    "outDir": "dist",
```
Comment (justinfagnani) [inline tsconfig.json:17]: dist? Do you want to convert this to our conventions?

**Review by justinfagnani — APPROVED** [review]

File: src/shady-css/common.ts (hunk 1 above)
Comment (rictic) [reply src/shady-css/common.ts]: +1, done

File: tsconfig.json (hunk 2 above)
Comment (rictic) [reply tsconfig.json:17]: Will do after this set of initial conversion PRs have landed, want to keep the changes approximately minimal.

**Pull request merged**